components/ipfs/resources/BUILD.gn

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2023 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import("//ui/webui/resources/tools/bundle_js.gni")
+
+# Generate JS bundle for RequestOTR interstitial UI, regradless of the
```

Comment:
s/RequestOTR/IPFS

Reply:
Fixed in f7478ff945177c88e0c9c65b7e42ae9b016b5345